SimCalorimetry/Configuration/python/ecalDigiSequence_cff.py

```diff
@@ -24,4 +24,26 @@
 from Configuration.Eras.Modifier_phase2_common_cff import phase2_common
 phase2_common.toReplaceWith(ecalDigiTask,_phase2_ecalDigiTask)
 
+from Configuration.Eras.Modifier_phase2_ecal_devel_cff import phase2_ecal_devel
+_phase2_ecalDigiTask_devel = cms.Task()
+phase2_ecal_devel.toReplaceWith(ecalDigiTask,_phase2_ecalDigiTask_devel)
 
+#phase 2 ecal pedestals                                                                                                                                                                                                                                                                  
+def _modifyEcalPedestals( process ):
+    process.load("SimCalorimetry.EcalSimProducers.esEcalLiteDTUPedestalsProducer_cfi")
+
+    process.load("CondCore.CondDB.CondDB_cfi")
+
+    process.CondDB.connect = 'sqlite_file:simPulseShapePhaseII.db'
```

Comment:
the connection string is still pointing to a local sqlite file

Reply:
Ah right right, stefano explained me how the DB works, i was confused. Sorry. I removed the call.